deep_quoridor/src/train_alphazero.py

```diff
@@ -268,6 +269,9 @@ def main(args):
         help="Every how many epochs to compute the benchmark",
     )
     parser.add_argument("--benchmarks_params", nargs="?", const="", default=None, type=str)
+    parser.add_argument(
+        "--benchmarks_skip_initial", action="store_true", default=False, help="Skip the initial benchmark"
+    )
```

Comment:
Can we use dashes (`--benchmarks-skip-initial`) instead of underscores for new arguments, since that's the posix convention? The python variable will end up as `benchmarks_skip_initial` automatically.